native-auth/config.md

```diff
@@ -9,3 +9,41 @@ dto NativeAuthClientConfig:
     gateway_url?: string;
     extra_request_headers?: dict[string, Any];
 ```
+
+## NativeAuthServerConfig
+
+```
+dto NativeAuthServerConfig:
+    // The endpoint from where the current block information will be fetched upon validation.
+    api_url: string;
+
+    // A mandatory list of accepted origins in case the server component must validate the incoming requests by domain.
```

Comment:
Here also maybe something like
`A required list of accepted origins for validating incoming request domains.`